cmd/kubeops/cmd/root.go

```diff
@@ -53,19 +47,19 @@ func init() {
 }
 
 func setFlags(c *cobra.Command) {
-	c.Flags().StringVar(&serveOpts.HelmDriverArg, "helm-driver", "", "which Helm driver type to use")
-	c.Flags().IntVar(&serveOpts.ListLimit, "list-max", 256, "maximum number of releases to fetch")
-	c.Flags().StringVar(&serveOpts.UserAgentComment, "user-agent-comment", "", "UserAgent comment used during outbound requests")
-	// Default timeout from https://github.com/helm/helm/blob/9fafb4ad6811afb017cc464b630be2ff8390ac63/cmd/helm/install.go#L146
-	c.Flags().Int64Var(&serveOpts.Timeout, "timeout", 300, "Timeout to perform release operations (install, upgrade, rollback, delete)")
 	c.Flags().StringVar(&serveOpts.ClustersConfigPath, "clusters-config-path", "", "Configuration for clusters")
 	c.Flags().StringVar(&serveOpts.PinnipedProxyURL, "pinniped-proxy-url", "http://kubeapps-internal-pinniped-proxy.kubeapps:3333", "internal url to be used for requests to clusters configured for credential proxying via pinniped")
 	c.Flags().StringVar(&serveOpts.PinnipedProxyCACert, "pinniped-proxy-ca-cert", "", "Path to certificate authority to use with requests to pinniped-proxy service")
 	c.Flags().IntVar(&serveOpts.Burst, "burst", 15, "internal burst capacity")
 	c.Flags().Float32Var(&serveOpts.Qps, "qps", 10, "internal QPS rate")
 	c.Flags().StringVar(&serveOpts.NamespaceHeaderName, "namespace-header-name", "", "name of the header field, e.g. namespace-header-name=X-Consumer-Groups")
 	c.Flags().StringVar(&serveOpts.NamespaceHeaderPattern, "namespace-header-pattern", "", "regular expression that matches only single group, e.g. namespace-header-pattern=^namespace:([\\w]+):\\w+$, to match namespace:ns:read")
-	c.Flags().StringVar(&serveOpts.GlobalReposNamespace, "global-repos-namespace", "kubeapps", "Namespace of global repositories")
+
+	// TODO(agamez): remove once a new version of the chart is released
+	err := c.Flags().MarkDeprecated("user-agent-comment", "(deprecated) UserAgent comment used during outbound requests")
```

Comment:
Because it is (as far as I can tell, -and the compiler is not complaining-) no longer being used since we removed all the unused logic (managing helm + managing repos). After this PR, kubeops is just serving the following operations: 1) listing all namespaces; 2) canI operation; 3) returning an opertor's logo.

Reply:
You are right, we still have the right user agent comment [here](https://github.com/vmware-tanzu/kubeapps/blob/main/chart/kubeapps/templates/apprepository/deployment.yaml#L88).